java/runtime/src/main/java/io/ray/runtime/io/BufferOutputStream.java

```diff
@@ -0,0 +1,24 @@
+package io.ray.runtime.io;
+
+import java.io.OutputStream;
+import java.nio.ByteBuffer;
+
+public class BufferOutputStream extends OutputStream {
+  private final MemoryBuffer buffer;
+
+  public BufferOutputStream(MemoryBuffer buffer) {
+    this.buffer = buffer;
+  }
+
+  public void write(int b) {
```

Comment:
It's the `java.io.OutputStream#write(int)` method in `java.io.OutputStream`, we can't set the type to int since we are overriding it. Just added all missing `@Override` annotations

Reply:
Got it.